<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="fourtek-desktop" ignored="0" name="Create Project Testcase" tests="6" failures="1" timestamp="18 Dec 2017 10:04:35 GMT" time="11.554" errors="0">
  <testcase name="startproject" time="4.112" classname="com.testcase.CreateProjectTest"/>
  <testcase name="typeProjectName" time="0.218" classname="com.testcase.CreateProjectTest"/>
  <testcase name="projectType" time="0.234" classname="com.testcase.CreateProjectTest"/>
  <testcase name="industryType" time="0.235" classname="com.testcase.CreateProjectTest"/>
  <testcase name="targetLanguage" time="0.277" classname="com.testcase.CreateProjectTest"/>
  <testcase name="doClickOnLetsStartButton" time="6.46" classname="com.testcase.CreateProjectTest">
    <failure type="java.lang.AssertionError" message="expected [Project created successfully!] but found [×
Project created successfully!]">
      <![CDATA[java.lang.AssertionError: expected [Project created successfully!] but found [×Project created successfully!]at com.testcase.CreateProjectTest.doClickOnLetsStartButton(CreateProjectTest.java:61)... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- doClickOnLetsStartButton -->
</testsuite> <!-- Create Project Testcase -->
